tests/generation/FileGlobs.py

```diff
@@ -100,15 +100,21 @@ def test_get_project_files_gitignore(self):
 
     def test_get_project_files_ci_mode(self):
         orig_cwd = os.getcwd()
-        os.chdir(os.path.dirname(os.path.realpath(__file__)) + 
```

Comment:
This should be a separate commit, and that commit should add a rule to .coafile so that this cant re-occur.
(Ideally a separate PR, as this one is already full)